tests/pytests/test.py

```diff
@@ -1124,20 +1127,23 @@ def testSlopInOrderIssue1986(env):
 def testExact(env):
     env.expect('ft.create', 'idx', 'ON', 'HASH',
                'schema', 'title', 'text', 'weight', 10.0, 'body', 'text').ok()
-    env.expect('ft.add', 'idx', 'doc1', 0.5, 'fields',
-               'title', 'hello world', 'body', 'lorem ist ipsum').ok()
-    env.expect('ft.add', 'idx', 'doc2', 1.0, 'fields',
-               'title', 'hello another world', 'body', 'lorem ist ipsum lorem lorem').ok()
-
-    res = env.cmd('ft.search', 'idx', '"hello world"', 'verbatim')
-    env.assertEqual(3, len(res))
-    env.assertEqual(1, res[0])
-    env.assertEqual("doc1", res[1])
+    env.cmd('HSET', '{doc}:1', 'title', 'hello world',
+            'body', 'lorem ist ipsum')
+    env.cmd('HSET', '{doc}:2', 'title', 'hello another world',
+            'body', 'lorem ist ipsum lorem lorem')
+
+    for dialect in range(1,6):
```

Comment:
Use `MAX_DIALECT` (and `set_max_dialect`)

Reply:
OK. `set_max_dialect()` now is part of `common.py`